docs/docs/experimentation-analysis/experiment-time-series.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Experiment Time Series
+slug: /app/experiment-time-series
+---
+
+# Experiment Time Series
+
+Time series showing the cumulative results of your experiment though a given experiment update are available in-line in the experiment results.
+
+![Experiment Time Series](/images/experiment-time-series.png)
+
+Each data point in the time series represents the cumulative results of the experiment as of a historical experiment update. There will be as many data points as there have been experiment updates, although we do some coarsening to one result per day when there are multiple results per day and many days in the experiment.
+
+![Tooltip of a single data point in the time series](/images/experiment-time-series-tooltip.png)
+
+Selecting a difference type (Relative, Absolute, Scaled Impact) will change the y-axis of the time series to the relevant effect. Selecting a different baseline variation will hide the time series as they are only computed for the default baseline variation.
+
+## FAQ
+
+### Why is some part of the time series dotted?
+
+The dotted part of the time series represents the results that were computed when the settings for either the metric or the analysis were changed. It's possible that results computed with the old settings do not line up with the current settings, so GrowthBook shows the part of the time series that is immediately comparable over time as a solid line, and uses a dotted line for the history where comparability is less certain.
```

Comment:
A dotted line in the time series indicates a period when metric settings or analysis settings were changed and historical results are. Because older data may not be directly comparable, the dotted line shows historical results where comparability is less certain. (A solid line line indicates results that are immediately comparable.)